packages/terra-notification-dialog/src/_ContentLayoutAsList.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNamesBind from 'classnames/bind';
+import ThemeContext from 'terra-theme-context';
+
+import styles from './ContentLayoutAsList.module.scss';
+
+const cx = classNamesBind.bind(styles);
+
+const propTypes = {
+  /**
+   * The list of strings to display in a list format.
+   */
+  items: PropTypes.arrayOf(PropTypes.string).isRequired,
+};
+
+const NotificationDialogContentList = ({ items }) => {
```

Comment:
Nit: I'd recommend we sync the export name and file name to whichever feels more clear.

Reply:
updated: https://github.com/cerner/terra-framework/pull/1105/commits/022584400b4aaafcf4ad53ff1ed2e2b2eaec08a5#diff-3b0ccb3029cb9edb23c8190ba85d2ff1R31